cli/commands/terraform/creds/providers/externalcmd/provider.go

```diff
@@ -43,7 +43,6 @@ func (provider *Provider) GetCredentials(ctx context.Context) (*providers.Creden
 		args = parts[1:]
 	}
 
-	provider.terragruntOptions.Logger.Infof("Executing %s to obtain Terragrunt credentials", provider.Name())
```

Comment:
Why did we lose this, @levkohimins ?

Reply:
Because it duplicates log:

```
time=2024-07-19T15:11:23+03:00 level=debug msg=Running command: echo -n {"awsCredentials": {"ACCESS_KEY_ID ....  
time=2024-07-19T15:11:23+03:00 level=debug msg=Command output will be suppressed. 
time=2024-07-19T15:11:23+03:00 level=debug msg=Obtaining AWS credentials from the external echo -n {"awsCredentials": {"ACCESS_KEY_ID ...
```